include/tudat/astro/basic_astro/hybridBodyShapeModel.h

```diff
@@ -0,0 +1,123 @@
+/*    Copyright (c) 2010-2022, Delft University of Technology
+ *    All rigths reserved
+ *
+ *    This file is part of the Tudat. Redistribution and use in source and
+ *    binary forms, with or without modification, are permitted exclusively
+ *    under the terms of the Modified BSD license. You should have received
+ *    a copy of the license with this file. If not, please or visit:
+ *    http://tudat.tudelft.nl/LICENSE.
+ *
+ */
+
+#ifndef TUDATBUNDLE_HYBRIDBODYSHAPEMODEL_H
```

Comment:
Could you change the include guard to TUDAT_.....?

Reply:
Done: https://github.com/MiguelAvillez/tudat/commit/fe74e9ab3c67ca5f33656faa39455d298a4796b5